deployments/container/Makefile

```diff
@@ -90,7 +87,6 @@ $(IMAGE_TARGETS): image-%:
 .PHONY: build
 build: $(DEFAULT_PUSH_TARGET)
 $(DEFAULT_PUSH_TARGET): build-$(DEFAULT_PUSH_TARGET)
-$(DEFAULT_PUSH_TARGET): DIST = $(DEFAULT_PUSH_TARGET)
```

Comment:
I noticed the following image was pushed via the GitHub Actions: `ghcr.io/nvidia/k8s-device-plugin:0ea49e43-`. The culprit is likely the `IMAGE_TAG` variable defined on line 32:
```
IMAGE_TAG ?= $(IMAGE_VERSION)-$(DIST)
```

Reply:
Good catch. I was rushing this a bit.